aten/src/ATen/native/layer_norm.cpp

```diff
@@ -278,12 +278,8 @@ Tensor rms_norm_symint(
         input.scalar_type(),
         "rms_norm",
         [&] {
-    scalar_t eps_val;
-    if (!eps.has_value()) {
-      eps_val = std::numeric_limits<at::scalar_value_type<scalar_t>::type>::epsilon();
-    } else {
-      eps_val = eps.value();
-    }
+    using limits =  std::numeric_limits<at::scalar_value_type<scalar_t>::type>;
```

Comment:
Lint would probably complains about a redundant space here I guess

Reply:
Surprisingly it doesn't.... But let me fix it anyway...